eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +88,68 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @param hash String "master"
+     * @return SHA of commit
+     * @throws JSONException if fails
+     * @throws IOException if fails
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha(final String hash) throws IOException, JSONException {
+        String sha = "master";
+        try {
+            final String query = String.format(
+                // @checkstyle LineLength (1 line)
+                "https://api.github.com/repos/objectionary/home/git/refs/heads/%s",
+                hash
+            );
+            final JSONObject obj = OyRemote.readJsonFromUrl(query);
+            sha = obj.getJSONObject("object").getString("sha");
+        } catch (final IOException | JSONException exception) {
+            Logger.info(this, exception.toString());
```

Comment:
@yasamprom I think two loggers is too much. Maybe it would be better to choose one of them?